app/models/file.ts

```diff
@@ -59,6 +60,9 @@ export default class FileModel extends BaseFileItem {
     @belongsTo('abstract-node', { polymorphic: true })
     target!: (AsyncBelongsTo<AbstractNodeModel> & AbstractNodeModel) | (AsyncBelongsTo<DraftNode> & DraftNode);
 
+    @hasMany('cedar-metadata-record', { inverse: null })
+    template!: AsyncBelongsTo<CedarMetadataRecordModel> & CedarMetadataRecordModel;
```

Comment:
I think according to the api contract the relationship name serialized through the API is `cedar_metadata_records`. So it should be `cedarMetadataRecords` here.

Reply:
I mean the name of the relationship, not the model.
```suggestion
    cedarMetadataRecords!: AsyncBelongsTo<CedarMetadataRecordModel> & CedarMetadataRecordModel;
```